python/core/auto_generated/gps/qgsgpsconnection.sip.in

```diff
@@ -114,6 +128,13 @@ Returns whether the connection information is valid
 Returns the fix status
 
 .. versionadded:: 3.10
+%End
+
+    QualityIndicator qualityIndicator() const;
```

Comment:
Can you deprecate int quality member, and add a new QualityIndicator mQualityIndicator member for storing this instead?